packages/n/niftiheader/xmake.lua

```diff
@@ -0,0 +1,19 @@
+package("niftiheader")
+    set_kind("library", {headeronly = true})
+    add_urls("https://nifti.nimh.nih.gov/pub/dist/src/nifti2/")
+    add_versions("latest", "")
+
+    on_download(function(package, opt)
+        import("net.http")
+        http.download(opt.url .. "nifti1.h", path.join(opt.sourcedir, "nifti1.h"))
+        http.download(opt.url .. "nifti2.h", path.join(opt.sourcedir, "nifti2.h"))
+    end)
```

Comment:
you can remove on_download and add_urls to. https://github.com/xmake-mirror/nifti2 mirror

Reply:
I don't understand, the link you posted returns 404 Not Found

Reply:
404?

<img width="1017" alt="image" src="https://user-images.githubusercontent.com/151335/162480438-db4d286a-fbc1-4d11-86e0-568010067763.png">

Reply:
```lua
add_urls("https://github.com/xmake-mirror/nifti2.git")
add_versions("0.0.1", "d2461b27888fd5db659edb8b788c7ce6203f703c")
```

then remove on_download

Reply:
```
$ git clone https://github.com/xmake-mirror/nifti2.git
Cloning into 'nifti2'...
remote: Repository not found.
fatal: repository 'https://github.com/xmake-mirror/nifti2.git/' not found
```
I don't seem to have access to this repo...

Reply:
oh, it is private indeed
![image](https://user-images.githubusercontent.com/17192695/162527035-3da345c2-2f0a-474d-b240-b26a6f3bceac.png)

Reply:
oh, sorry, I have made it as public.